ManagerRegistry.php

```diff
@@ -20,11 +20,40 @@
 
 namespace Doctrine\Bundle\PHPCRBundle;
 
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+use Symfony\Component\DependencyInjection\ContainerInterface;
 use Symfony\Bridge\Doctrine\ManagerRegistry as BaseManagerRegistry;
 use Doctrine\ODM\PHPCR\PHPCRException;
 
 class ManagerRegistry extends BaseManagerRegistry
 {
+    public function __construct(
```

Comment:
this is technically a BC break, but we are moving to version 2 so thats ok.

i think we should mark this class as  `@internal` , users should just use it in the service but not as custom code.

Reply:
made the class final and marked it as internal

Reply:
okay, making it final does not work out. reverted that bit, only left the comment